pkg/cloudinit/cloudconfig.go

```diff
@@ -18,9 +18,11 @@ import (
 )
 
 type CloudConfig struct {
-	Apt            Apt                `yaml:"apt,omitempty"`
-	Packages       []string           `yaml:"packages,omitempty"`
-	Runcmd         [][]string         `yaml:"runcmd,omitempty"`
+	Apt           Apt      `yaml:"apt,omitempty"`
+	Packages      []string `yaml:"packages,omitempty"`
+	PackageUpdate bool     `yaml:"package_update,omitempty"`
+	// Runcmd could be either []string or string
```

Comment:
cloud-init support both []string and string. but I think just use []string or string is a better idea, because go would enforce the type during unmarshal.